R/ppc-distributions.R

```diff
@@ -17,9 +17,44 @@
 #'   the predictive distributions.
 #' @param ... For dot plots, optional additional arguments to pass to [ggdist::stat_dots()].
 #'
+#' @param binwidth Numeric scalar; passed to the underlying geom/stat used
+#'   by the plotting function. Behavior by function:
+#'   * `ppc_hist()`: forwarded to `ggplot2::geom_histogram(binwidth = ...)`.
+#'   * `ppc_freqpoly()` / `ppc_freqpoly_grouped()`: forwarded to
+#'     `ggplot2::geom_area(stat = "bin", binwidth = ...)` (i.e. `stat = "bin"`).
+#'   * `ppc_dots()`: forwarded to `ggdist::stat_dots(binwidth = ...)`.
+#'   If `NULL`, the default binning behavior of the respective geom/stat is used.
+#'
+#' @param bins Integer number of bins. Forwarded in the same way as `binwidth`
+#'   (i.e. to `geom_histogram()`, `geom_area(stat = "bin")`, and `ggdist::stat_dots()`).
+#'
+#' @param breaks Numeric vector of break positions. Forwarded in the same way
+#'   as `binwidth`/`bins` (where supported).
+#'
+#' @param freq Logical. Controls whether the vertical scale shows counts
+#'   (`freq = TRUE`) or densities/proportions (`freq = FALSE`). This argument
+#'   is not limited to `ppc_hist()` — it is also used by `ppc_freqpoly()` and
+#'   `ppc_freqpoly_grouped()` because those functions use `stat="bin"`
+#'   (implemented via `geom_area(stat = "bin")`). The `freq` value is passed
+#'   to `set_hist_aes()` and affects the y aesthetic used by `geom_histogram()`
+#'   and `geom_area(stat="bin")`.
+#'
 #' @template details-binomial
 #' @template return-ggplot-or-data
 #'
+#' @section Version & Dependencies:
+#'
+#' - Requires **ggplot2 (>= 3.4.0)** for the use of modern layering and
+#'   consistent handling of bin arguments.
+#'
+#' - Uses **ggdist::stat_dots()** (requires **ggdist >= 3.3.0**).
+#'
+#' - All `ppc_*` functions depend on **bayesplot** core plotting infrastructure
+#'   and `ggplot2` grammar of graphics.
+#'
+#' - Ensure that dependent packages (`ggplot2`, `ggdist`, and `bayesplot`)
+#'   are loaded or properly imported in your package’s `DESCRIPTION` file.
```

Comment:
This is all true, but it should already be enforced by the DESCRIPTION file we have. That is, you can't install bayesplot without ggplot2 >= 3.4.0. Although I do notice that we're missing the ggdist version number, thanks for pointing that out! 

I'm not sure all this text is worth adding to the documentation page since we would need to add a section like this to every R file in the package for consistency and this information is already included in the DESCRIPTION file.